canonical-webteam/generator-canonical-webteam
# Add version information to all generated files

For the run script generated stuff.

This will allow us to tell how up-to-date all parts of a project are.

QA
--

``` bash
sudo npm run link  # Use the version of this module from this branch
```

Now go over to a project that makes use of `./run`, and regenerate things:

``` bash
yo canonical-webteam:run
```

Check that it adds versioning to files it touches correctly.
